src/test/java/ParkingTollUtilsTest.java

```diff
@@ -0,0 +1,90 @@
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+import utils.ParkingTollUtils;
+
+public class ParkingTollUtilsTest {
+
+	
+	@Test
+	public void givenHoursNumber_whenCalcualteVIPFee_thenReceiveFeeForGivenHour(){
+		assertEquals(0, ParkingTollUtils.getVIPDriverTollInHour(1), 0.1);

```

Comment:
każda z tych asercji to odzielny przypadek testowy i powinien być w osobnym teście - ewentualnie testy sparametryzowane